src/vs/code/electron-sandbox/processExplorer/processExplorerMain.ts

```diff
@@ -86,7 +91,7 @@ class ProcessExplorer {
 
 		// Format name with indent
 		const formattedName = isRoot ? name : `${'    '.repeat(indent)} ${name}`;
-		const memory = process.platform === 'win32' ? item.mem : (totalmem() * (item.mem / 100));
+		const memory = this.data.platform === 'win32' ? item.mem : (totalmem() * (item.mem / 100));
```

Comment:
should this use `getTotalmem` instead of `totalmem` ?

Reply:
Yes, thanks!